### ISSUE TITLE
Show link to the linked domain after deploy.

### ISSUE THREAD

**Issue by krutilin** [reimagined/resolve-cloud#63]
It is good to have link to the domain bonded to the application after deploy complete. It is useful to click and go. (see Zeit\Vercel's deploy UX for reference)

We have:
```
✔ "brewcomputer-app-1" available at https://f7j5qvinadaaqneg4z1lpj2xnuv1.resolve.sh      
```

I suppose it should be:
```
✔ "brewcomputer-app-1" available at: 
https://f7j5qvinadaaqneg4z1lpj2xnuv1.resolve.sh
https://stage.brewcomputer.com
```

**Comment by max-vasin** [comment]
Hi, thanks for good proposal. We'll take it into account.

**Comment by MrCheater** [comment]
`yarn resolve-cloud list --format="{{ applicationName }}, {{deploymentId}}, {{applicationUrl}}"`
Fixed in version 0.28.2
